src/components/SecondaryPanes/Frames.css

```diff
@@ -47,13 +47,23 @@
   margin-right: 1em;
 }
 
-.frames ul li:hover,
+.frames ul li:hover {
+  background-color: rgba(236, 236, 236,0.6);
```

Comment:
We could call this something like: `--theme-selectable-background-hover` which should indicate that is a select-able object.